Source Code/Doubly Linked List (Insert, Delete, Print).c

```diff
@@ -273,3 +280,21 @@ void printLinkedListBackward()
     }
     puts("\n");
 }
+
+//This function search the value in the list
+int search(int value)
```

Comment:
Can we rename it like `isValueAvailable(int value)`? Because this method returns the availability of the value.
Also you can set the return type as `bool` and return `true` or `false` from the method.

Reply:
Yeah Sure